jupyterlab/commands.py

```diff
@@ -240,6 +239,12 @@ def _write_config(data, config_dir=None):
     """Write the JupyterLab config data.
     """
     config_dir = config_dir or _get_config_dir()
+    if not osp.exists(config_dir):
+        try:
+            os.makedirs(config_dir)
+        except OSError as e:
+            if e.errno != errno.EEXIST:
+                pass
```

Comment:
this does nothing in any case right? did you want to do something if the errno is or isn’t EEXIST?